xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3691,17 +3691,13 @@ bool CVideoPlayer::OpenVideoStream(CDVDStreamInfo& hint, bool reset)
   if (hint.flags & AV_DISPOSITION_ATTACHED_PIC)
     return false;
 
-  // set desired refresh rate
-  if (m_playerOptions.fullscreen && CServiceBroker::GetWinSystem()->GetGfxContext().IsFullScreenRoot() &&
-      hint.fpsrate != 0 && hint.fpsscale != 0)
+  // Set refresh rate, at reset request (playback start)
```

Comment:
Does reset happen only on playback request? If not I'd remove the comment to avoid confusion and keep it as it was before.

Reply:
done